docs/mint.json

```diff
@@ -777,7 +777,8 @@
             "integrations/app-integrations/slack",
             "integrations/app-integrations/twitter",
             "integrations/app-integrations/web-crawler",
-            "integrations/app-integrations/youtube"
+            "integrations/app-integrations/youtube",
+            "integrations/app-integrations/mediawiki"
```

Comment:
Let's list it in alphabetical order. So `"integrations/app-integrations/mediawiki"` would come after `"integrations/app-integrations/hackernews",`.